### DISCUSSION TITLE
Add easy way to disable animation in sign up form

### DISCUSSION THREAD

**Discussion by tom93** [community/community#84732] [Accessibility] [answered]
### Select Topic Area

Product Feedback

### Body

The sign-up form (<https://github.com/signup>) has an animated background of moving stars. The animation is beautiful, but it is very distracting and can induce motion sickness (not to mention that it is CPU intensive).

I see that the page respects the "prefers-reduced-motion" CSS media feature; thank you for that.

But I don't want to disable animations globally.

Please add an easy way to disable animations just on the GitHub sign-up page; for example a small button in the top right corner of the "Welcome to GitHub!" box.

**Comment by ghostinhershell** [comment]
Hi @tom93, 

Thanks so much for reaching out and including as much information as you did. I've submitted this feedback over to the Accessibility team so it's now on their radar. If you have any additional comments or feedback that you'd like to share, please feel free to do so here and I can pass that along as well.

Thanks!

**Comment by hans-djalali** [comment]
This extension does exist, but it has a limitation: it only functions properly when you don't resize or scroll the page. So, basically, its usefulness is somewhat very limited, but it's still worth trying out and sharing. Link: [Stop Animations Extension](https://chromewebstore.google.com/detail/stop-animations/gemmknjnneiojfjelmgappppbaneikda).

**Comment by cehfisher** [comment]
Hello @tom93 - thank you for taking the time to provide this feedback! Our Accessibility team is actively working on this issue. We will follow-up here with any additional questions or updates. Thanks again!

**Comment by cehfisher** [comment, marked as answer]
Hello @tom93 - we have some great news to share on this issue. As of this week, the moving stars have been removed from the sign-up form completely! This should help you and countless others who reported sensitivity to this motion. Let us know if you have any follow-up questions. Thanks!
